src/wui/interactive_player.cc

```diff
@@ -450,10 +478,10 @@ void InteractivePlayer::draw_map_view(MapView* given_map_view, RenderTarget* dst
 			// Draw build help.
 			bool show_port_space = has_expedition_port_space(f->fcoords);
 			if (show_port_space || buildhelp()) {
-				const auto* overlay = get_buildhelp_overlay(
-				   show_port_space ? f->fcoords.field->maxcaps() : plr.get_buildcaps(f->fcoords));
-				if (overlay != nullptr) {
-					blit_field_overlay(dst, *f, overlay->pic, overlay->hotspot, scale);
+				if (const auto* overlay = get_buildhelp_overlay(
+				       show_port_space ? f->fcoords.field->maxcaps() : plr.get_buildcaps(f->fcoords))) {
+					blit_field_overlay(dst, *f, overlay->pic, overlay->hotspot, scale,
+					                   f->seeing == Widelands::SeeUnseeNode::kVisible ? 1.f : 0.4f);
```

Comment:
```suggestion
					                   f->seeing == Widelands::SeeUnseeNode::kVisible ? 1.f : 0.3f);
```

Code LGTM, but I think the difference isn't noticeable enough yet. It would pop even better at `0.2f`, but then the port space icon would be too hard to see. SO, I think `0.3f` is our best option.